project/src/components/place-card-list/place-card-list.tsx

```diff
@@ -0,0 +1,20 @@
+import {useState} from 'react';
+import { NO_CARD_ID } from '../../const';
+import { Offers } from '../../types/offer';
+import PlaceCard from '../place-card/place-card';
+
+type PlaceCardListProps = {
+  offers: Offers;
+}
+
+export default function PlaceCardList({offers}: PlaceCardListProps) {
+  const [activeCardId, setActiveCardId] = useState(NO_CARD_ID);
```

Comment:
Изначальное состояние следует указать null, также если пока activeCardId то его можно не доставать
```suggestion
  const [, setActiveCardId] = useState<number | null>(null);
```